caddy/command.go

```diff
@@ -0,0 +1,228 @@
+package caddy
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/caddyserver/caddy/v2"
+	"github.com/caddyserver/caddy/v2/caddyconfig"
+	caddycmd "github.com/caddyserver/caddy/v2/cmd"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp/encode"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp/fileserver"
+	"github.com/caddyserver/caddy/v2/modules/caddyhttp/rewrite"
+	"github.com/caddyserver/certmagic"
+	"go.uber.org/zap"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	caddycmd.RegisterCommand(caddycmd.Command{
+		Name:  "php-server",
+		Usage: "[--domain <example.com>] [--root <path>] [--listen <addr>] [--access-log]",
+		Short: "Spins up a production-ready PHP server",
+		Long: `
+A simple but production-ready PHP server. Useful for quick deployments,
+demos, and development.
+
+The listener's socket address can be customized with the --listen flag.
+
+If a domain name is specified with --domain, the default listener address
+will be changed to the HTTPS port and the server will use HTTPS. If using
+a public domain, ensure A/AAAA records are properly configured before
+using this option.
+
+For more advanced use cases, see https://github.com/dunglas/frankenphp/blob/main/docs/config.md.`,
```

Comment:
:sweat_smile: I tried to click this in github but the extra dot at the end made the link fail to work